ocaml/testsuite/tests/polling/polling.ml

```diff
@@ -0,0 +1,27 @@
+(* TEST
+ {
+   poll-insertion;
+   toplevel.opt;
+ }
+*)
+
+let[@poll error] loop_alloc x =
+  for _ = 0 to x do
+    ignore(Sys.opaque_identity(ref 42))
+  done
+;;
+
+let[@poll error] loop_no_alloc x =
+  for _ = 0 to x do
+    ()
```

Comment:
One day we might be able to detect that this loop can be deleted, so I would put something like `ignore (Sys.opaque_identity 42)` in the middle of it.